frontend/src/components/GitlabDashboard/GitlabProfile.js

```diff
@@ -0,0 +1,233 @@
+                    },
+                }}
+            />
+            <br></br>
+            <Select
+                options={selectProject}
+                value={project}
+                placeholder="Select Project"
+                onChange={(params) => {
+                    setProject(params.value);
+                    console.log("value", project);
+                    if (project.length !== 0) {
+                        let idx = parseInt(project[0].id);
+                        console.log("idx", idx, project[0].label);
+                        let pageToSet = Math.ceil(
+                            (idx + 1) / props.postsPerPage
+                        );
+                        console.log(pageToSet);
+                        setCurrentPage(pageToSet);
+                    }
+                }}
+                size={SIZE.compact}
+                overrides={{
+                    ControlContainer: {
+                        style: ({ $theme }) => ({
+                            borderRadius: "4px",
+                        }),
+                    },
+                }}
+            />
+            <br></br>
+            {/* <Select
+                options={[
+                    { label: "AliceBlue", id: "#F0F8FF" },
+                    { label: "AntiqueWhite", id: "#FAEBD7" },
+                    { label: "Aqua", id: "#00FFFF" },
+                    { label: "Aquamarine", id: "#7FFFD4" },
+                    { label: "Azure", id: "#F0FFFF" },
+                    { label: "Beige", id: "#F5F5DC" },
+                ]}
+                value={mergeRequest}
+                placeholder="Select mergeRequest"
+                onChange={(params) => setMergeRequest(params.value)}
+            /> */}
+            {/* <Table
+                columns={[
+                    "ProjectName",
+                    "Merge-Request",
+                    "Merge-Status",
+                    "Reviewer",
+                    "Target-Branch",
+                    "Approval-Status",
+                    "Pipeline-Status",
+                ]}
+                data={currentpost}
+                className="gitlab"
+            />
+            <br></br>
+            <br></br>
+            <br></br>
+            <div id="pagination">
+                <Pagination
+                    numPages={totalpages}
+                    currentPage={currentPage}
+                    onPageChange={({ nextPage }) => {
+                        setCurrentPage(Math.min(Math.max(nextPage, 1), 20));
+                    }}
+                    className="gitlab"
+                /> */}
+
+            {/* </div> */}
+            <Table
+                data={currentpost}
+                pageNumber={currentPage}
+                totalPages={Math.ceil(props.post.length / props.postsPerPage)}
+                setPageNumber={setCurrentPage}
+                loading={false}
+                columnTitles={[
+                    "ProjectName",
```

Comment:
don't keep titles as camel case. rename to "Project Name", "Merge Request", "Status", Target Branch etc.